### PR TITLE
Serialized empty List to []

### LINKED ISSUE #1: Empty list encoding seems incorrect
```dart
final convertedEmptyList = YAMLWriter().convert([]);
```

In the above example the converted empty list is an empty String, I would expect "[]".

### PR DESCRIPTION
**Pull request by rorystephenson**
Previously it serialized to an empty String which means it would be deserialized as a null instead of an empty List.

Fixes #1.

### CODE REVIEW

**Comment by gmpassos** [comment]
Please, run a `dart format .` in your branch and commit the changes to complete the CI checks.

Grazie per la correzione.

File: test/yaml_writer_test.dart [hunk 1]
```diff
@@ -52,6 +52,39 @@ void main() {
       expect(yamlWriter.convert(tree), equals(yaml));
     });
 
+    test('empty list', () {
```
Comment (gmpassos) [inline test/yaml_writer_test.dart:55]: Please run `dart format .` and commit the changes to fix the CI checks.

**Comment by rorystephenson** [comment]
> Please, run a `dart format .` in your branch and commit the changes to complete the CI checks.
> 
> Grazie per la correzione.

Fatto! Grazie per il package :).

**Comment by gmpassos** [comment]
Grazie per la correzione!

**Comment by gmpassos** [comment]
I have published a new version:

v1.0.2

https://pub.dev/packages/yaml_writer

**Pull request merged**